mammoth/opts.py

```diff
@@ -339,65 +274,17 @@ def model_opts(parser):
         action='store_true',
         help="Use a sin to mark relative words positions. Necessary for non-RNN style models.",
     )
-    group.add(
-        "-update_vocab", "--update_vocab", action="store_true", help="Update source and target existing vocabularies"
-    )
-
-    group = parser.add_argument_group('Model-Embedding Features')
-    group.add(
-        '--feat_merge',
-        '-feat_merge',
-        type=str,
-        default='concat',
-        choices=['concat', 'sum', 'mlp'],
-        help="Merge action for incorporating features embeddings. Options [concat|sum|mlp].",
-    )
-    group.add(
-        '--feat_vec_size',
-        '-feat_vec_size',
-        type=int,
-        default=-1,
-        help="If specified, feature embedding sizes "
-        "will be set to this. Otherwise, feat_vec_exponent "
-        "will be used.",
-    )
-    group.add(
-        '--feat_vec_exponent',
-        '-feat_vec_exponent',
-        type=float,
-        default=0.7,
-        help="If -feat_merge_size is not set, feature "
-        "embedding sizes will be set to N^feat_vec_exponent "
-        "where N is the number of values the feature takes.",
-    )
-
-    # Model Task Options
-    group = parser.add_argument_group("Model- Task")
-    group.add(
-        "-model_task",
-        "--model_task",
-        default=ModelTask.SEQ2SEQ,
-        choices=[ModelTask.SEQ2SEQ, ModelTask.LANGUAGE_MODEL],
-        help="Type of task for the model either seq2seq or lm",
-    )
 
     # Encoder-Decoder Options
     group = parser.add_argument_group('Model- Encoder-Decoder')
-    group.add(
-        '--model_type',
-        '-model_type',
-        default='text',
-        choices=['text'],
-        help="Type of source model to use. Allows the system to incorporate non-text inputs. Options are [text].",
-    )
     group.add('--model_dtype', '-model_dtype', default='fp32', choices=['fp32', 'fp16'], help='Data type of the model.')
 
     group.add(
         '--encoder_type',
         '-encoder_type',
         type=str,
         default='transformer',
-        choices=['mean', 'transformer'],
+        choices=['mean', 'transformer'],  # TODO is this mean actually supported?
```

Comment:
No. Attempting to use it results in `AssertionError: Only Transformer is supported`